lib/builderator/interface/packer.rb

```diff
@@ -138,12 +138,20 @@ def _chef_provisioner_base
           :environments_path => Config.local.environment_path,
           :chef_environment => Config.profile.current.chef.environment,
           :json => Config.profile.current.chef.node_attrs,
+          :staging_directory => Config.chef.staging_directory,
+          :execute_command => _chef_execute_command(true),
+          :install_command => _chef_install_command(true)
```

Comment:
So the issue is the container itself?  Because we actually create the staging directory with the shell provisioner before running the chef provisioner.  There has to be a staging directory specified somehow, and from what I recall, it's different version of chef that cause it to move.

What do you think is the right way to fix this, such that building AMIs works by specifying the staging directory (as it has since at least 97997bc48070f63c67bbe5b53e1f03889131c5e7 in the form of `Config.chef.staging_directory`) and also that we can "just let packer and chef do their things"?